### ISSUE TITLE
"Go to Tag" and "Move To Tag" keybinds are not working. 

### ISSUE THREAD

**Issue by 2brownc** [leftwm/leftwm#918]
### Describe the bug

1. Going to a tag `n` using `modkey + n` is not working.
2. Moving a window to a tag `n` using `modkey + Shit + n` is not working.

### Expected behavior (optional)

1. Going to a tag `n` using `modkey + n` should work.
2. Moving a window to a tag `n` using `modkey + Shit + n` should work.

### Steps to reproduce (optional)

**Go to  Tag**

1. Be at tag, for example, 1.
2. Press `modkey + 2`.
3. Does show tag 2.

**Move to Tag**

1. Open a terminal with `modkey + Shift + Return` in tag 1.
2. Press `modkey + Shift + 2`.
3. Terminal in tag 1 does not move to tag 2.

### OS / Distro

Debian 5.10.149-1 (2022-10-17) x86_64

### LeftWM Check

```text
:: LeftWM version: 0.4.0
:: LeftWM git hash: NONE
:: Loading configuration . . .
    -> Configuration loaded OK 
:: Checking keybinds . . .
    -> All keybinds OK
:: Checking environment . . .
    -> Environment OK 
:: Checking theme . . .
WARN: TOML as config format is about to be deprecated.
      Please consider migrating to RON or contact the theme creator about this topic.
      Note: make sure the `up` script is loading the correct theme file.
ERROR: File not found: theme.ron
```

**Comment by VuiMuich** [comment]
This might be related to a bad `config.toml`.
If don't have too many deviating config options from the default, could you please consider moving your old config to a backup location (like renaming it to `config.tom_backup`) and run `leftwm-check` to generate a new  `config.ron`. Then recreate your customized options manually as described in the [migration guide](https://github.com/leftwm/leftwm/wiki/Migrate-Toml-To-Ron) in the wiki.

**Comment by 2brownc** [comment]
I removed previous config (which was barely touched) and ran `leftwm-check`. It did create a new configuration and I even restarted the system. But the issue persists, I cannot switch tags or move windows to different tags with keybindings.

- I installed `leftwm` using `cargo`.
- I am using the Ascent theme. Is it the theme?
- I did not touch the config file. 

The config file: https://pastebin.com/cRELmsVH
